StatisticsInR/2.1_simple_linear_regression.Rmd

```diff
@@ -8,62 +8,97 @@ output:
     css: ../css/style-chapters.css
 ---
 
-```{r setup-template, eval=FALSE, include=TRUE}
+```{r setup, eval=TRUE, include=FALSE}
```

Comment:
In the template it says to leave the title blank as it might appear with the table of contents L69 of the template .Rmd @audhalbritter which should it be? I just want to be consistent with the other pages. But leaving blank does give a warning.

Reply:
Not adding a title was before we changed to bookdown. But Jonathan says it does not make a difference now. Let's just use titles, to avoid the warnings. I'll change this in the template!